### ISSUE TITLE
Libraries to look out for

### ISSUE THREAD

**Issue by andreipfeiffer** [andreipfeiffer/css-in-js#8]
- [x] Compiled: https://github.com/atlassian-labs/compiled/issues/531 (ADDED)
- [ ] Jess: https://jesscss.github.io/ (currently in alpha)
- [x] vanilla-extract: https://github.com/seek-oss/vanilla-extract (ADDED)
- [ ] microsoft/@Griffel: https://griffel.js.org/react/guides/ssr-usage
- [ ] twind: https://twind.dev/
- [ ] StyleX: https://stylex-docusaurus-nmn.vercel.app/ (currently in beta)

**Comment by itsdouges** [comment]
Heya! Compiled now has an extraction story, you can read more here:

- https://compiledcssinjs.com/docs/css-extraction-webpack
- https://twitter.com/compiledcss/status/1373125637762772996?s=20

**Comment by andreipfeiffer** [comment]
@madou thanks for the heads up, great to hear that you've managed to ship this experimental feature.

1. CSS static extraction is opt-in, so you'll support both approaches, right?
2. If it's not too much to ask, could you please ping me when you have a stable version/implementation?

**Comment by itsdouges** [comment]
1. Yep!
2. Sure.

**Comment by azu** [comment]
📝 vanilla-extract v1.0 is released.
https://twitter.com/markdalgleish/status/1398158090294292482

**Comment by andreipfeiffer** [comment]
Hey @azu thanks for the update. I saw the release announcement :)
I already created the example, I just need to update it to the latest version and write the review.

**Comment by olegshilov** [comment]
Good day!
I think this is a good candidate to be added to the libraries list
https://github.com/4Catalyzer/astroturf
Postcss has mentioned this lib in readme  [here](https://github.com/postcss/postcss#css-in-js)

**Comment by andreipfeiffer** [comment]
Hi @olegshilov,
Astroturf has already been considered, but not included in the analysis:
https://github.com/andreipfeiffer/css-in-js#astroturf

There was a Next.js example, which we didn't manage to start.
However, that example is not available anymore, as far as I can see.

**Comment by dawidk92** [comment]
@andreipfeiffer also you could add [Stitches](https://stitches.dev/) library

**Comment by andreipfeiffer** [comment]
@dawidk92 
Stitches is already added https://github.com/andreipfeiffer/css-in-js#stitches. 
The only thing I need to do is to update it to [v1](https://stitches.dev/blog/stitches-1.0.0), which was recently released.

**Comment by dawidk92** [comment]
@andreipfeiffer awesome :) also you can add it to the Overview section

**Comment by andreipfeiffer** [comment]
It should be there or am I missing something?

<img width="651" alt="Screenshot 2021-09-09 at 12 14 29" src="https://user-images.githubusercontent.com/2570562/132668688-2e19eef2-ea5f-4735-b2a5-a91289ba6d31.png">

**Comment by dawidk92** [comment]
Yup @andreipfeiffer, sorry for a misconception. I'm blind I guess 🙈

**Comment by andreipfeiffer** [comment]
No worries, it happens to all of us, and to me as well 😁

**Comment by dawidk92** [comment]
Maybe alphabetical order would do the trick for blind people like me 😄 I was looking at this table and I couldn't see it. I don't know how it's possible 😄

**Comment by tienlx97** [comment]
microsoft/@Griffel is css-in-js/ts use atomic-css. Please check.

**Comment by andreipfeiffer** [comment]
Hi @lexuantien thanks for pointing it out, will consider it. Are you a maintainer of the library?

**Comment by redbar0n** [comment]
[twind.dev](https://twind.dev/) is Tailwind made into CSS-in-JS / CSS-in-TS (it ships a runtime Tailwind compiler, not the CSS...).

**Comment by andreipfeiffer** [comment]
Hey @redbar0n, twind looks interesting. I'll add it to the list.

**Comment by athammer** [comment]
stylex just launched https://stylex-docusaurus.vercel.app/

**Comment by mryechkin** [comment]
> stylex just launched https://stylex-docusaurus.vercel.app/

Prod link: https://stylexjs.com

Looks interesting. I'm currently evaluating it against [Panda CSS](https://panda-css.com/) to replace Styled Components in my lib.

**Comment by nikelborm** [comment]
a few more
https://yak.js.org/
https://www.restyle.dev/
